app/scripts/modules/core/src/modal/wizard/WizardModal.tsx

```diff
@@ -153,9 +155,15 @@ export class WizardModal<T extends FormikValues> extends React.Component<IWizard
     setErrors(this.validate(values));
   }
 
+  private setWaiting(section: string, isWaiting: boolean): void {
+    const waiting = new Set(this.state.waiting);
+    isWaiting ? waiting.add(section) : waiting.delete(section);
+    this.setState({ waiting });
```

Comment:
is the intention to only ever have a max of 1 waiting section? not clear on why there's a `Set` if it's only ever going to have 1 thing in it.

Reply:
nvm cannot read, disregard.